components/header.tsx

```diff
@@ -55,29 +68,29 @@ export default function Header() {
           alignItems: "center",
         }}
       >
-        <Link href="/" passHref>
+        <Link activeClassName='active' href="/" passHref>
```

Comment:
what are you trying to do here? I thought you were going to create a react component in order to handle this situation, what happened with that idea?